extensions/functions_arithmetic.yaml

```diff
@@ -742,6 +742,52 @@ aggregate_functions:
           - value: fp64
         nullability: DECLARED_OUTPUT
         return: fp64?
+  - name: "corr"
+    description: Calculates correlation of two set of values.
```

Comment:
This should also be specifying that the function returns null if there is no input (I'm assuming from context that that's how it works, anyway)

Reply:
You mean to add it to the description?

Reply:
The descriptions should document how the functions behave, including corner cases, yes. And likewise for mode, which has even less obvious corner cases.

Reply:
Updated the description.